src/main/java/edu/depaul/cdm/se452/d2l_mock/course/CoursePage.java

```diff
@@ -0,0 +1,60 @@
+package edu.depaul.cdm.se452.d2l_mock.course;
+
+import edu.depaul.cdm.se452.d2l_mock.course.Announcements.Announcement;
+import edu.depaul.cdm.se452.d2l_mock.course.Announcements.AnnouncementRepository;
+import edu.depaul.cdm.se452.d2l_mock.course.Events.Event;
+import edu.depaul.cdm.se452.d2l_mock.course.Events.EventRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
```

Comment:
Do we need this file for Milestone2 which only relates to persistence?

Reply:
This was from MS1